onnxruntime/contrib_ops/cuda/custom_layer_norm.cu

```diff
@@ -0,0 +1,305 @@
+/* Modifications Copyright (c) Microsoft. */
```

Comment:
copyright statement seems a little odd since there's nothing about the original (before the modifications)

Reply:
Modified